src/assets/scripts/_resources-utils.js

```diff
@@ -225,9 +225,9 @@ export function renderSearchResults(numberOfResources, resourceTopics, resourceT
     let appliedFilterHtml = "<h2>Search results</h2><div class='resources-applied-filters'>";
 
     if (numberOfResources === 0) {
-        appliedFilterHtml += "<div class='resources-no-results' role='alert'><p>Sorry, no results were found based on your applied filters.</p></div>";
+        appliedFilterHtml += "<div class='resources-no-results' id='resources-sort-result'><p>Sorry, no results were found based on your applied filters.</p></div>";
```

Comment:
I would change this to `resources-result` as it's relevant for both sorting and filtering.